### ISSUE TITLE
[portal] "ensure_next()" is broken

### ISSUE THREAD

**Issue by ericsnowcurrently** [faster-cpython/tools#103]
At the least it's broken with the "compare" command.  (See https://github.com/faster-cpython/tools/pull/102#discussion_r972463730.)

```
$ bench-jobs-dev compare req-compile-bench-1663116583-esnow req-compile-bench-1663116597-esnow
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 192, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__main__.py", line 1177, in <module>
    main(cmd, cmd_kwargs, cfgfile, user, devmode)
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__main__.py", line 1126, in main
    jobs.ensure_next()
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/__init__.py", line 272, in ensure_next
    queue = self.queue.snapshot
  File "/home/esnow/faster-cpython-tools/PORTAL/jobs/queue.py", line 246, in snapshot
    assert pid is not None
AssertionError
```

It may be broken under other commands.

**Comment by ericsnowcurrently** [comment]
CC @mdboom

**Comment by mdboom** [comment]
Fixed in #109.  I think this was just an incorrect assert.
